engine/source/output/anim/generate/nodalssp.F

```diff
@@ -0,0 +1,115 @@
+Chd|====================================================================
+Chd|  NODALP                        source/output/anim/generate/nodalp.F
+Chd|-- called by -----------
+Chd|        GENANI                        source/output/anim/generate/genani.F
+Chd|        H3D_NODAL_SCALAR              source/output/h3d/h3d_results/h3d_nodal_scalar.F
+Chd|-- calls ---------------
+Chd|        ANINFLOW                      source/output/anim/generate/aninflow.F
+Chd|        ELBUFDEF_MOD                  ../common_source/modules/elbufdef_mod.F
+Chd|        I22BUFBRIC_MOD                ../common_source/modules/interfaces/cut-cell-search_mod.F
+Chd|        I22TRI_MOD                    ../common_source/modules/interfaces/cut-cell-search_mod.F
+Chd|        INITBUF_MOD                   share/resol/initbuf.F         
+Chd|====================================================================
+      SUBROUTINE NODALSSP(IFUNC    , WA4      , WA4_FVM, IFLOW  , RFLOW,  
+     .                    IPARG    , ELBUF_TAB, IX     , NIX    , NUMEL, 
+     .                    ITAB     , NV46     , MONVOL , VOLMON , AIRBAGS_TOTAL_FVM_IN_H3D,
+     .                    IS_WRITTEN_NODE,IS_WRITTEN_NODE_FVM, ISPMD, FVDATA_P, SWA4)
+C-----------------------------------------------
+C   D e s c r i p t i o n
+C-----------------------------------------------
+C   This subroutine defines & writes nodal sound speed requested by Engine keyword /H3D/NODA/SSP
+C      application :  /MONVOL/FVMBAG* (sound spped from polyhedra, specific buffer).
+C-----------------------------------------------
+C   P r e - C o n d i t i o n s
+C-----------------------------------------------
+C     none
+C-----------------------------------------------
+C   M o d u l e s
+C-----------------------------------------------
+      USE INITBUF_MOD
+      USE ELBUFDEF_MOD  
+      USE FVBAG_MOD , only:FVBAG_DATA,NFVBAG  
+      USE GROUPDEF_MOD , only:GROUP_      
+C-----------------------------------------------
+C   I m p l i c i t   T y p e s
+C-----------------------------------------------
+#include      "implicit_f.inc"
+C-----------------------------------------------
+C   C o m m o n   B l o c k s
+C-----------------------------------------------
+#include      "com01_c.inc"
+#include      "com04_c.inc"
+#include      "vect01_c.inc"
+#include      "param_c.inc"
+#include      "tabsiz_c.inc"
+C-----------------------------------------------
+C   D u m m y   A r g u m e n t s
+C-----------------------------------------------
+      INTEGER,INTENT(IN) :: NUMEL, IFUNC, NIX, NV46,ITAB(NUMNOD), ISPMD, SWA4
+      INTEGER,INTENT(IN) :: IFLOW(*),IPARG(NPARG,NGROUP),IX(NIX,NUMEL)
+      my_real,INTENT(IN) :: RFLOW(*)
+      INTEGER, INTENT(IN) :: AIRBAGS_TOTAL_FVM_IN_H3D
+      REAL,INTENT(INOUT) :: WA4(SWA4), WA4_FVM(AIRBAGS_TOTAL_FVM_IN_H3D)
+      TYPE (ELBUF_STRUCT_), INTENT(IN), DIMENSION(NGROUP), TARGET :: ELBUF_TAB    
```

Comment:
Dummy argument cannot have the TARGET attribute unless there is an explicit interface.
If possible I suggest to remove the TARGET attribute since it seems to be used only for aliasing (replace GBUF by the full name).
If it cannot be done, another solution would be to encapsulate the subroutine in a module

Reply:
TARGET arrtibute removed. thanks